composer.json

```diff
@@ -13,7 +13,7 @@
     "require": {
         "php": ">=5.3.0",
         "fsi/datasource" : "~1.2",
-        "ruflin/elastica": "~1.2",
+        "ruflin/elastica": "~2.3",
```

Comment:
tests for --prefer-lowest are also required before 1.0.0 release and at least 1.0.0-rc tag in order to update dependencies in those project and check if everything works fine with ruflin/elastica 2.3.

Also I think it would be good to change ~2.3 to ^2.4